### PR TITLE
Add installation instructions

### PR DESCRIPTION
**Pull request by vrabaud**
This should fix https://github.com/AOMediaCodec/libavif/issues/2081

### CODE REVIEW

**Review by y-guyon — APPROVED** [review]
Thanks. Maybe Windows pre-built binaries section could be moved next to windows intall section?

**Comment by vrabaud** [comment]
I thought the pre-built Windows binaries were at HEAD so I purposefully left it in the dev section. But it's not the case so moving it.

**Review by wantehchang — APPROVED** [review]

**Comment by kmilos** [comment]
Does it also make sense to mention libavif packages are also available in [MSYS2 (Windows)](https://packages.msys2.org/base/mingw-w64-libavif) and [MacPorts](https://ports.macports.org/port/libavif/details/)?

**Comment by vrabaud** [comment]
@kmilos sure, what are the instructions to install on those platforms? Thx

**Comment by kmilos** [comment]
`pacman -S mingw-w64-ucrt-x86_64-libavif` (for the "default" MSYS2 UCRT64 environment; need to adapt the package triplet obviously for other target environments)

`sudo port install libavif` I guess for MacPorts...

**Comment by kmilos** [comment]
Btw, you could also add a "packaging status" like [libheif](https://github.com/strukturag/libheif?tab=readme-ov-file#packaging-status) and [libjxl](https://github.com/libjxl/libjxl?tab=readme-ov-file#installation) do 😉

**Comment by vrabaud** [comment]
I always thought those were noisy. @wantehchang , do you want to add https://repology.org/badge/vertical-allrepos/libavif.svg?exclude_unsupported=1&columns=3&exclude_sources=modules,site&header=libheif%20packaging%20status ?

**Comment by kmilos** [comment]
Should be https://repology.org/badge/vertical-allrepos/libavif.svg?exclude_unsupported=1&columns=3&exclude_sources=modules,site&header=libavif%20packaging%20status (header should match)

File: README.md [hunk 1]
```diff
@@ -48,6 +53,13 @@ Red Hat-based distributions:
 sudo yum -y install libavif
 ```
 
+### MinGW
```
Comment (wantehchang) [inline README.md:56]: Nit: MinGW should be moved to the "Windows" section, at least next to it.

**Review by wantehchang — COMMENTED** [review]
LGTM. Thank you (and Miloš) for adding the package installation instructions!

**Comment by wantehchang** [comment]
Vincent: I think it's good to add the [libavif packaging status](https://repology.org/badge/vertical-allrepos/libavif.svg?exclude_unsupported=1&columns=3&exclude_sources=modules,site&header=libavif%20packaging%20status). I agree that table may grow too big, and it is more useful to us (libavif maintainers) than the end users. We can address that by setting up the link so that it opens a new page.

**Pull request merged**